.github/workflows/gardener_issue_comment.yml

```
@@ -26,44 +26,43 @@ jobs:
           status_field_id="PVTF_lADOAQFeYs4AAsTrzgAXRuU" # Status
           triage_option_id="2a08fafa"
 
-          # ensures that the issue is already on board but also seems to be the only way to fetch
-          # the item id
-          item_id="$(gh api graphql -f query='
-            mutation($project_id: ID!, $content_id: ID!) {
-                addProjectV2ItemById(input: {projectId: $project_id, contentId: $content_id}) {
-                  item {
-                    id
-                  }
-                }
-            }' -f project_id="$project_id" -f content_id="$issue_id" -q '.data.addProjectV2ItemById.item.id'
-          )"
-
-          echo "item_id: $item_id"
-
-          if [ -z "$item_id" ] ; then
-            echo "Issue not found in Gardener board"
-            exit 0
-          else
-            echo "Found issue on Gardener board"
-          fi
-
-          current_status="$(gh api graphql -f query='
+          # Query for project items for the given issue
+          project_items="$(gh api graphql -f query='
             query($item_id: ID!) {
               node(id: $item_id) {
-                ... on ProjectV2Item {
-                  fieldValueByName(name: "Status") {
-                    ... on ProjectV2ItemFieldSingleSelectValue {
-                      name
+                ... on Issue {
+                  projectItems(first: 10) {
```

```suggestion
                  projectItems(first: 50) {
```

We already have 8 projects so I could see us accidentally reaching this limit without realizing this needs to be updated. You can also see an approach for automatic pagination to fetch all nodes that you could use, here:

https://gist.github.com/jszwedko/e0fc6a6000497d7df810d4eb9051b7d5#file-copy-cards-js-L142-L204

---

Neat, I've already spent too much time on this, so I'm going to simply bump the page size. It's extremely unlikely for a single issue to be linked to 50 projects.